hack/local-up-cluster.sh

```diff
@@ -1027,6 +1027,7 @@ EOF
       --feature-gates="${FEATURE_GATES}" \
       --authentication-kubeconfig "${CERT_DIR}"/scheduler.kubeconfig \
       --authorization-kubeconfig "${CERT_DIR}"/scheduler.kubeconfig \
+      --leader-elect=false \
```

Comment:
why is that?

Reply:
it defaults to false so technically a noop. I was just trying to be consistent so we're explicit in stating leader election conditions for local-up-cluster.sh script. Happy to remove